cmd/clusterctl/cmd/generate_cluster.go

```diff
@@ -137,6 +139,7 @@ func init() {
 	// other flags
 	generateClusterClusterCmd.Flags().BoolVar(&gc.listVariables, "list-variables", false,
 		"Returns the list of variables expected by the template instead of the template yaml")
+	generateClusterClusterCmd.Flags().StringVarP(&gc.output, "output", "o", "-", "Specify the output file to write the template to, defaults to `-` for STDOUT")
```

Comment:
What about using another flag name? output / -o for everyone in the Kubernete ecosystem pretty much stands for (kubectl)
```
    -o, --output='':
        Output format. One of: (json, yaml, name, go-template, go-template-file, template, templatefile, jsonpath,
        jsonpath-as-json, jsonpath-file, custom-columns, custom-columns-file, wide). See custom columns
        [https://kubernetes.io/docs/reference/kubectl/#custom-columns], golang template
        [http://golang.org/pkg/text/template/#pkg-overview] and jsonpath template
        [https://kubernetes.io/docs/reference/kubectl/jsonpath/].
```

I think it would be confusing to have the same flag with different behavior

@fabriziopandini WDYT?

Reply:
I have a suggestion:
-O (for the shorthand)
--to (the long one)
@sbueringer @fabriziopandini suggestions?

Reply:
I think write-to is okay (I don't have a better idea). I wouldn't assign a shorthand. We don't necessarily do this for all flags, mostly only to the ones very frequently used.

Reply:
done. @sbueringer PTAL.

Reply:
Thx!

Reply:
I don't have strong preferences on --output or --write-to (ok to --write-to)
Just wondering if we should not default to "-" and just use stout if the flag is missing

Reply:
Either should be fine IMO, but let me know if it's preferable to use stdout if the flag is missing.